intuita-webview/src/errors/App.tsx

```diff
@@ -60,14 +72,17 @@ const buildExecutionErrorRow = (
 declare global {
 	interface Window {
 		INITIAL_STATE: {
-			viewProps: View['viewProps'];
+			communityProps: CommunityViewProps;
+			errorsViewProps: ErrorsViewProps;
+			codemodRunsProps: CodemodRunsViewProps;
+			fileExplorerProps: FileExplorerViewProps;
 		};
 	}
 }
 
 export const App = () => {
-	const [viewProps, setViewProps] = useState<ViewProps>(
-		window.INITIAL_STATE.viewProps as ViewProps,
+	const [viewProps, setViewProps] = useState<ErrorsViewProps>(
+		window.INITIAL_STATE.errorsViewProps as ErrorsViewProps,
```

Comment:
is the type cast necessary any longer? I think your changes have ensured we don't need the casting anymore